src/Monolog/Handler/MongoDBHandler.php

```diff
@@ -28,14 +28,20 @@
 class MongoDBHandler extends AbstractProcessingHandler
 {
     protected $mongoCollection;
+    protected $namespace;
+    protected $manager;
 
     public function __construct($mongo, $database, $collection, $level = Logger::DEBUG, $bubble = true)
     {
-        if (!($mongo instanceof \MongoClient || $mongo instanceof \Mongo || $mongo instanceof \MongoDB\Client)) {
+        if (!($mongo instanceof \MongoClient || $mongo instanceof \Mongo || $mongo instanceof \MongoDB\Client || $mongo instanceof \MongoDB\Driver\Manager)) {
             throw new \InvalidArgumentException('MongoClient, Mongo or MongoDB\Client instance required');
         }
-
-        $this->mongoCollection = $mongo->selectCollection($database, $collection);
+        $this->namespace = "$database.$collection";
```

Comment:
The namespace is only applicable for the `MongoDB\Driver\Manager` case, so we can probably move this within the conditional below. Typically, `$database . '.' . $collection` is preferred over variable name expansion in double-quoted strings.